diff --git a/node_modules/tree-sitter-perl/binding.gyp b/node_modules/tree-sitter-perl/binding.gyp
index ee2fa0c..7ad33e9 100644
--- a/node_modules/tree-sitter-perl/binding.gyp
+++ b/node_modules/tree-sitter-perl/binding.gyp
@@ -11,7 +11,7 @@
       "sources": [
         "bindings/node/binding.cc",
         "src/parser.c",
-        # NOTE: if your language has an external scanner, add it here.
+        "src/scanner.c",
       ],
       "conditions": [
         ["OS!='win'", {
